slackroll

```diff
@@ -1250,6 +1303,10 @@ def verify_local_names(names, local_list):
             sys.exit('ERROR: %s is not a local package name' % name)
 
 def post_kernel_operation(): # Operations needed after kernel installation
+    if slackroll_batch_mode:
+        print 'WARNING: You will need to handle the bootloader configuration and rebooting manually'
```

Comment:
Fine but I'd word this more carefully, using "may" instead of "will". For example:

Warning: you may need to modify your bootloader configuration and reboot.

Reply:
Updated